### PR TITLE
[docs] Add missing library logos in API reference

### PR DESCRIPTION
**Pull request by amandeepmittal**
# Why

Some of the libraries API references are missing logos. However, for these libraries, the logo is already provided in each of their README files.

# How

Add the missing logo for 
- Cellular
- Clipboard
- Contacts
- Device
- Document Picker
- Font
- Keep Awake
- Location
- Print
- Screen Orientation
- Sharing
- Speech
- Splash Screen
- SQLite
- Status Bar
- Store Review
- Updates

Changes backported to for all SDK versions currently available and unversioned.

**Example preview** after these changes

CleanShot 2023-10-07 at 17 14 52@2x

CleanShot 2023-10-07 at 17 15 21@2x

CleanShot 2023-10-07 at 17 15 31@2x

CleanShot 2023-10-07 at 17 15 37@2x

# Test Plan

- Run docs locally
- Visit a library doc from the above list
- Check different SDK versions to verify that logo is displayed for that library 

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/versions/unversioned/sdk/speech.mdx [hunk 1]
```diff
@@ -3,6 +3,7 @@ title: Speech
 description: A library that provides access to text-to-speech functionality.
 sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-speech'
 packageName: 'expo-speech'
+iconUrl: '/static/images/packages/expo-sharing.png'
```
Comment (tsapeta) [inline docs/pages/versions/unversioned/sdk/speech.mdx:6]:
```suggestion
iconUrl: '/static/images/packages/expo-speech.png'
```

File: docs/pages/versions/v47.0.0/sdk/speech.mdx [hunk 2]
```diff
@@ -3,6 +3,7 @@ title: Speech
 description: A library that provides access to text-to-speech functionality.
 sourceCodeUrl: 'https://github.com/expo/expo/tree/sdk-47/packages/expo-speech'
 packageName: 'expo-speech'
+iconUrl: '/static/images/packages/expo-sharing.png'
```
Comment (tsapeta) [inline docs/pages/versions/v47.0.0/sdk/speech.mdx:6]:
```suggestion
iconUrl: '/static/images/packages/expo-speech.png'
```

File: docs/pages/versions/v48.0.0/sdk/speech.mdx [hunk 3]
```diff
@@ -3,6 +3,7 @@ title: Speech
 description: A library that provides access to text-to-speech functionality.
 sourceCodeUrl: 'https://github.com/expo/expo/tree/sdk-48/packages/expo-speech'
 packageName: 'expo-speech'
+iconUrl: '/static/images/packages/expo-sharing.png'
```
Comment (tsapeta) [inline docs/pages/versions/v48.0.0/sdk/speech.mdx:6]:
```suggestion
iconUrl: '/static/images/packages/expo-speech.png'
```

**Review by tsapeta — APPROVED** [review]
Thank you! 💙

File: docs/pages/versions/v49.0.0/sdk/speech.mdx [hunk 4]
```diff
@@ -3,6 +3,7 @@ title: Speech
 description: A library that provides access to text-to-speech functionality.
 sourceCodeUrl: 'https://github.com/expo/expo/tree/sdk-49/packages/expo-speech'
 packageName: 'expo-speech'
+iconUrl: '/static/images/packages/expo-sharing.png'
```
Comment (amandeepmittal) [inline docs/pages/versions/v49.0.0/sdk/speech.mdx:6]:
```suggestion
iconUrl: '/static/images/packages/expo-speech.png'
```

**Pull request merged**